tasks/src/image-segmentation/data.ts

```diff
@@ -59,6 +59,10 @@ const taskData: TaskDataCustom = {
 			description: "A semantic segmentation application that predicts human silhouettes.",
 			id:          "keras-io/Human-Part-Segmentation",
 		},
+		{
+			description: "Cell Instance Segmentation - Sartorius Kaggle Competition",
```

Comment:
Can you give more simpler context on what this is about? You can take a look at the other definitions 🙂